images/virtualization-artifact/pkg/controller/cvi/internal/source/http.go

```diff
@@ -100,6 +105,13 @@ func (ds HTTPDataSource) Sync(ctx context.Context, cvi *virtv2.ClusterVirtualIma
 
 		log.Info("Cleaning up...")
 	case pod == nil:
+		ds.recorder.Event(
```

Comment:
Either add a start event for everyone, or remove.

Reply:
Added for everyone